api/network/load_balancers_api.go

```diff
@@ -28,7 +28,7 @@ func NewLoadBalancerService(concertoService utils.ConcertoService) (*LoadBalance
 func (lb *LoadBalancerService) GetLoadBalancerList() (loadBalancers []types.LoadBalancer, err error) {
 	log.Debug("GetLoadBalancerList")
 
-	data, status, err := lb.concertoService.Get("/v1/network/load_balancers")
+	data, status, err := lb.concertoService.Get("/v2/network/load_balancers")
```

Comment:
The same applies to load balancers, they are not an available resource either...